lamindb_setup/core/_settings_instance.py

```diff
@@ -56,6 +56,8 @@ def __init__(
         schema: str | None = None,  # comma-separated string of schema names
         git_repo: str | None = None,  # a git repo URL
         is_on_hub: bool | None = None,  # initialized from hub
+        api_url: str | None = None,
+        schema_id: UUID | None = None,
```

Comment:
@falexwolf not sure. I have to add them here for saving the instance settings. But i am not sure if they should be here even as just private variables.

Reply:
I think this is OK, Sergei!